### PR TITLE
Add fallbacks and prompt to master.py

### PR DESCRIPTION
**Pull request by dannyqiu**
- Handles empty lines (by pressing enter)
- Gracefully handles bad `pid` argument
- Handles empty command with valid `pid`

### CODE REVIEW

File: master.py [hunk 1]
```diff
@@ -134,14 +134,20 @@ def main(debug=False):
     timeout_thread.start()
 
     while True:
+        sys.stdout.write("> ")
```
Comment (ccding) [inline master.py]: please print it only in debug mode; otherwise it will mess up the grading script

File: master.py [hunk 2]
```diff
@@ -131,17 +131,24 @@ def timeout():
 def main(debug=False):
     global threads, wait_ack
     timeout_thread = Thread(target=timeout, args=())
+    timeout_thread.setDaemon(True)
```
Comment (ccding) [inline master.py]: why do you want to use daemon mode in master?

File: master.py [hunk 3]
```diff
@@ -150,12 +157,16 @@ def main(debug=False):
...
 
         if sp1[0] == 'sleep':  # sleep command
             time.sleep(int(sp1[1]) / 1000)
             continue
 
-        pid = int(sp2[0])  # first field is pid
+        try:
+            pid = int(sp2[0])  # first field is pid
+        except ValueError:
+            continue
```
Comment (ccding) [inline master.py]: print an error message here

File: master.py (hunk 2 above)
Comment (dannyqiu) [reply master.py]: The timeout thread prevents master from closing on `Ctrl+C` or when master raises an exception. Setting it to daemon allows master to exit, which is already what it does after 60 seconds

File: master.py (hunk 1 above)
Comment (dannyqiu) [reply master.py]: Removing this entirely, since it's only good for manually typing in commands

**Comment by ccding** [comment]
LGTM

**Pull request merged**